platform/mbed_alloc_wrappers.cpp

```diff
@@ -48,7 +48,15 @@ typedef struct {
 
 #ifdef MBED_HEAP_STATS_ENABLED
 static SingletonPtr<PlatformMutex> malloc_stats_mutex;
-static mbed_stats_heap_t heap_stats = {0, 0, 0, 0, 0};
+static mbed_stats_heap_t heap_stats = {0, 0, 0, 0, 0, 0, 0};
+
+typedef struct  {
+    size_t size;
+}mbed_heap_overhead_t;
```

Comment:
Does this apply to all allocators or just one particular library? dlmalloc based?

Reply:
Does not apply to all allocators. But tested with dlmalloc and all three tool chains

Reply:
Which from Mbed OS point of view is "all"

Thanks..